README.md

```diff
@@ -20,10 +20,6 @@ if let Some((Width(w), Height(h))) = size {
 }
 ```
 
-## Minimum Rust Version
-
-This crate requires a minimum rust version of 1.63.0 (2022-08-11)
```

Comment:
Without additional scripting, there's not an easy way to duplicate this in the README.md.  I'd argue it's not necessarily and anyone interested will hopefully now know to look for `rust-version`.

That said...it seems to me that crates.io, lib.rs and docs.rs etc. should render this in their HTML views too.

Reply:
I keep my MSRV to a schedule and use renovatebot to update it: https://github.com/clap-rs/clap/blob/master/.github/renovate.json5#L29

The problem I've found is if you don't do this,
- people will assume your MSRV is something it isn't 
- contributors will be shy about doing or asking for MSRV bumps

Alternatively, before I had that tool, I had a tag in every location for updating it (effectively what I match the regex on now).

One way to keep CI in sync is if you use `cargo hack <cmd> --rust-version`.